app/main.py

```diff
@@ -1,3 +1,65 @@
+from typing import Union
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: Union["Distance", float, int]) -> "Distance":
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        else:
+            return Distance(self.km + other)
+
+    def __radd__(self, other: Union[float, int]) -> "Distance":
+        return self.__add__(other)
+
+    def __iadd__(self, other: Union["Distance", float, int]) -> "Distance":
+        if isinstance(other, Distance):
+            self.km += other.km
+        else:
+            self.km += other
+        return self
+
+    def __mul__(self, other: Union[float, int]) -> "Distance":
+        return Distance(self.km * other)
+
+    def __truediv__(self, other: Union[float, int]) -> "Distance":
+        return Distance(round(self.km / other, 2))
+
+    def __lt__(self, other: Union["Distance", float, int]) -> bool:
+        if isinstance(other, Distance):
+            return self.km < other.km
+        else:
```

Comment:
You don’t need `else` if you have a `return` statement in `if`. Just call the second terurnt on the same lvl as `if` statement

Reply:
@dsokolovskiy 
I see you corrected the code in this place, what about other places where the same error occurs?